README.md

```diff
@@ -13,18 +13,18 @@ from a Youtube Channel/Username or one-or-more Playlists.
 
 It downloads the video (`webm` or `mp4` extension – optionally
 recompress them in lower-quality, smaller size), the thumbnails, the
-subtitles and the authors' profile pictures ; then, it create a static
-HTML files folder of it before creating a ZIM off of it.
+subtitles and the authors' profile pictures ; then produces JSON files
+containing content for the channel, playlists, and videos, which are then utilized by a Vue.js UI.
 
 `youtube2zim` adheres to openZIM's [Contribution Guidelines](https://github.com/openzim/overview/wiki/Contributing).
 
-`youtube2zim` has implemented openZIM's [Python bootstrap, conventions and policies](https://github.com/openzim/_python-bootstrap/docs/Policy.md) **v1.0.2**.
+`youtube2zim` has implemented openZIM's [Python bootstrap, conventions and policies](https://github.com/openzim/_python-bootstrap/blob/main/docs/Policy.md) **v1.0.2**.
 
 Requirements
 ------------
 
 * [`ffmpeg`](https://ffmpeg.org/) for video transcoding (only used with `--lower-quality`).
```

Comment:
ffmpeg is now always used, the `--low-quality` only switches between high and low presets. We should remove the "(only used with ...)" statement.

Reply:
Removed in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).